projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -931,7 +929,7 @@
       }
 
       &.datagrid-column {
-        padding-top: calc(tables-variables.$clr-table-compact-vertical-padding + tokens.$cds-global-space-1);
+        padding-top: tables-variables.$clr-table-compact-vertical-padding;
         padding-bottom: tables-variables.$clr-table-compact-vertical-padding;
```

You can use shorter version. 
```suggestion
        padding-block: tables-variables.$clr-table-compact-vertical-padding 0;
```

---

We are not using `padding-block` anywhere using it only here migth lead to some inconsistencies.